src/features/home/Home.tsx

```diff
@@ -29,19 +30,26 @@ const types = [
   {id: 3, type: TypesDish.Snacks},
 ];
 
+@inject(Stores.DishStore)
+@observer
 export class Home extends React.Component<Props, State> {
   public state: State = {
     currentType: TypesDish.Foods,
+    refresh: false,
   };
 
+  public async componentDidMount() {
+    await this.props.dish.fetchDishes();
+  }
+
   public render() {
     return (
       <ScrollView style={styles.container}>
         <View style={styles.header}>
           <TouchableOpacity onPress={this.hangleOpenDrawer}>
             <Image source={require('../../assets/image/Vector.png')} />
           </TouchableOpacity>
-          <TouchableOpacity>
+          <TouchableOpacity onPress={this.navigateOrders}>
             <Image source={require('../../assets/image/orders.png')} />
           </TouchableOpacity>
```

Comment:
@DiamondRock597 Buttons component with children or with image that can receive src from props.